ci/docker/Dockerfile.build.android_arm64

```diff
@@ -18,23 +18,20 @@
 #
 # Dockerfile to build MXNet for Android ARM64/ARMv8
 
-FROM ubuntu:16.04 as ccachebuilder
-
-COPY install/ubuntu_core.sh /work/
-RUN /work/ubuntu_core.sh
-COPY install/ubuntu_ccache.sh /work/
-RUN /work/ubuntu_ccache.sh
+#FROM ubuntu:16.04 as ccachebuilder
+#COPY install/ubuntu_core.sh /work/
+#RUN /work/ubuntu_core.sh
+#COPY install/ubuntu_ccache.sh /work/
+#RUN /work/ubuntu_ccache.sh
 
 FROM dockcross/base:latest
```

Comment:
Why not use dockcross/android-arm64 ?

Reply:
Happy to do so in a subsequent PR, now this one passed. I would very much like to get it merged so at least we have this build as a gatekeeping mechanism.

Reply:
Ok, cool